ports/libnpy/vcpkg.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "libnpy",
```

```suggestion
  "name": "matajoh-libnpy",
```
It might be clearer to use matajoh-libnpy instead. Currently, both Repology and Google searches for libnpy refer to llohse/libnpy, not the one introduced in this PR, which could lead to confusion.